java/Nikiforov-Oleksandr/src/com/zoolatech/lecture7/tasks/_2/Calculator.java

```diff
@@ -0,0 +1,21 @@
+package com.zoolatech.lecture7.tasks._2;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public abstract class Calculator {
```

Comment:
Do you really need to have this base class? You're overriding it's methods in all three subclasses, and for the atomic version the value even stored in a different variable.